code/lib/core-server/src/utils/warnOnIncompatibleAddons.ts

```diff
@@ -6,6 +6,10 @@ import dedent from 'ts-dedent';
 import { getIncompatibleAddons } from '../../../cli/src/automigrate/helpers/getIncompatibleAddons';
 
 export const warnOnIncompatibleAddons = async (config: StorybookConfig) => {
+  if (config.features?.warnOnIncompatibleAddons === false) {
```

Comment:
@shilman I'm guessing `warnOnIncompatibleAddons` isn't temporary, so you'd want to locate it somewhere else?

Reply:
Hey @taozhou-glean thanks for the contribution. However I have a question:

@ndelangen @shilman why should there be such feature like this? We can gracefully handle the warnOnIncompatibleAddons function instead. If users have a broken experience because of an internal JsPackageManager issue, and later have to somehow figure out that they can set up a flag to disable it, it seems weird to me. If we just handle the error, it will work for everyone. This code path (though important for users migrating) isn't critical anyway